content/en/docs/reference/glossary/feature-gates.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Feature gates
+id: feature-gates
+date: 2023-01-12
+full_link: /docs/reference/command-line-tools-reference/feature-gates/
+short_description: >
+  Feature gates are a set of key=value pairs that describe Kubernetes features.
+
+aka: 
+tags:
+- fundamental
+- operation
+---
+
+Feature gates are a set of key=value pairs that describe Kubernetes features.
```

Comment:
```suggestion
Feature gates are a set of keys (opaque string values) that you can use to control which
Kubernetes features are enabled in your cluster.
```

Reply:
The values (enabled or disabled) aren't part of the feature gate. For example, `IndexedJob` is a feature gate you can enable; `IndexedJob=true` isn't.

Reply:
@sftim, 
Thanks for the review comments. I incorparated the suggestions. PTAL!!